examples/nidcpower/measure-record.py

```diff
@@ -153,7 +147,7 @@ def on_close(event):
         closed = True
     fig.canvas.mpl_connect('close_event', on_close)
 
-    x_axis = np.arange(start=1, stop=record_length+1, step=1)
+    x_axis = np.arange(start = 1, stop = record_length + 1, step = 1)
```

Comment:
This line is actually no longer required since we setup x_axis during every iteration of loop on line 188 below. (verify once by running the example w/o this line).

Reply:
Yes, you are right, I forgot to remove it. After removing also its working as expected.